docs/src/main/asciidoc/microprofile-health.adoc

```diff
@@ -124,10 +124,10 @@ public class SimpleHealthCheck implements HealthCheck {
     }
 }
 ----
+<1> It's recommended to annotate the health check class with `@ApplicationScoped` or the `@Singleton` scope so that a single bean instance is used for all health check requests. 
+<2> If a bean class annotated with one of the health check annotations declares no scope then the `@Singleton` scope is used automatically.
 
-As you can see health check procedures are defined as implementations of the 
-`HealthCheck` interface which are defined as CDI beans with the one of the 
-following CDI qualifiers: 
+As you can see, the health check procedures are defined as CDI beans that implementat the `HealthCheck` interface and are annotated with the one of the health check qualifiers, such as: 
```

Comment:
"implementat" should be "implement"?

Reply:
Maybe change "with the one of" to "with one of"

Reply:
Yep, sure. Thanks for spotting these!